drush/README.md

```diff
@@ -0,0 +1,3 @@
+The three subdirectories here hold project-specific commandfiles, site aliases, and configuration for Drush. These will all be included automatically when you use the ./dr wrapper script that should be in your project root directory. The dr script intentionally skips loading commands/aliases/config from the typical global locations in order to ensure that all team members have same Drush environment.
+ 
+ See http://packages.drush.org/ for a directory of Drush commands installable via Composer.
```

Comment:
Ah yes, you are correct.

I also notice that packagist.drupal-composer.org (or, more accurately, the parser that builds component info from drupal.org projects) and packagist.org (which is to say, what everyone seems to be entering into their composer.json files) are both standardized on the type 'drupal-drush' now, which is good.  For a while, some folks were using 'drush-extension', but that usage has thankfully fallen into disuse.